### ISSUE TITLE
PlannerConfiguration: Det skjedde en feil under konfigurering av området

### ISSUE THREAD

**Issue by zetard** [Puzzlepart/prosjektportalen365#292]
**Describe the bug**
When running wizard after creating a project-site the wizard throws error: 

PlannerConfiguration: Det skjedde en feil under konfigurering av området

**To Reproduce**

1. Go to 'https://tenant.sharepoint.com/sites/projectURL/'
2. Click on 'Sett opp området' with default values
3. See error

**Expected behavior**
Wizard finishes

**Screenshots**
![image](https://user-images.githubusercontent.com/20675654/88386639-1e54c980-cdb1-11ea-9bd0-5bc0133e7244.png)

**About your setup**

- Browser Chrome, IE, Edge (Chromium)
- Installed version 1.2.2

**Additional context**
Just installed the package in a dev tenant. Got some warnings during installation: 
![image](https://user-images.githubusercontent.com/20675654/88387208-4f81c980-cdb2-11ea-9d36-d1592527a9cd.png)

**Comment by olemp** [comment]
Hi @zetard. Can you find additional error information in the console window?

**Comment by zetard** [comment]
First i got a message that the group "Porteføljeadministrator" did not exist on the parent site

VM65644:1 GET https://TENANT.sharepoint.com/sites/prosjektportalen/_api/web/sitegroups/getByName('Portef%C3%B8ljeadministrator')

I created it manually and that logmessage went away. First line after starting the "oppsett av prosjektområde" is this: 

VM65644:1 GET https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Prosjektegenskaper') 404 (Not Found)

This i correct, list does not exist. 

VM65644:1 POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Kommunikasjonsplan')/fields('%7Bd685f33f-51b5-4e9f-a314-4b3d9467a7e4%7D') 400 (Bad Request)
Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {d685f33f-51b5-4e9f-a314-4b3d9467a7e4}  /sites/test4/Lists/Kommunikasjonsplan"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}
VM65644:1 POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Prosjektlogg')/fields('%7B022cc93f-13df-4420-bd47-55e4fdae5d18%7D') 400 (Bad Request)
Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {022cc93f-13df-4420-bd47-55e4fdae5d18}  /sites/test4/Lists/Prosjektlogg"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}
POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Prosjektleveranser')/fields('%7B6d90e0b6-73e6-48fb-aa1e-b897b214f934%7D') 400 (Bad Request)
Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {6d90e0b6-73e6-48fb-aa1e-b897b214f934}  /sites/test4/Lists/Prosjektleveranser"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}
POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Gevinstanalyse%20og%20gevinstrealiseringsplan')/fields('%7B1d5752af-4d26-4aed-b20a-6229ac14ed5d%7D') 400 (Bad Request)
Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {1d5752af-4d26-4aed-b20a-6229ac14ed5d}  /sites/test4/Lists/Gevinstanalyse og gevinstrealiseringsplan"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}
POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('M%C3%A5leindikatorer')/fields('%7B8d70fa93-b547-46f1-84e7-4982f8c9c675%7D') 400 (Bad Request)
Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {8d70fa93-b547-46f1-84e7-4982f8c9c675}  /sites/test4/Lists/Mleindikatorer"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}

Basically the same message with different list and fields goes on for about 60 lines after that. Then:

spfxsinglesignon.aspx:28 Cannot read property 'getRequestInfo' of null
project-setup-application-customizer_480ec34efa912f888af570b2bc34fca5.js:26 Message: (ProjectSetup) [_initializeSetup]: Failed

**Comment by olemp** [comment]
> First i got a message that the group "Porteføljeadministrator" did not exist on the parent site
> 
> `VM65644:1 GET https://TENANT.sharepoint.com/sites/prosjektportalen/_api/web/sitegroups/getByName('Portef%C3%B8ljeadministrator')`
> 
> I created it manually and that logmessage went away. First line after starting the "oppsett av prosjektområde" is this:
> 
> `VM65644:1 GET https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Prosjektegenskaper') 404 (Not Found) `
> This i correct, list does not exist.
> 
> ```
> VM65644:1 POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Kommunikasjonsplan')/fields('%7Bd685f33f-51b5-4e9f-a314-4b3d9467a7e4%7D') 400 (Bad Request)
> Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {d685f33f-51b5-4e9f-a314-4b3d9467a7e4}  /sites/test4/Lists/Kommunikasjonsplan"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}
> VM65644:1 POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Prosjektlogg')/fields('%7B022cc93f-13df-4420-bd47-55e4fdae5d18%7D') 400 (Bad Request)
> Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {022cc93f-13df-4420-bd47-55e4fdae5d18}  /sites/test4/Lists/Prosjektlogg"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}
> POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Prosjektleveranser')/fields('%7B6d90e0b6-73e6-48fb-aa1e-b897b214f934%7D') 400 (Bad Request)
> Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {6d90e0b6-73e6-48fb-aa1e-b897b214f934}  /sites/test4/Lists/Prosjektleveranser"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}
> POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('Gevinstanalyse%20og%20gevinstrealiseringsplan')/fields('%7B1d5752af-4d26-4aed-b20a-6229ac14ed5d%7D') 400 (Bad Request)
> Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {1d5752af-4d26-4aed-b20a-6229ac14ed5d}  /sites/test4/Lists/Gevinstanalyse og gevinstrealiseringsplan"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}
> POST https://TENANT.sharepoint.com/sites/test4/_api/web/lists/getByTitle('M%C3%A5leindikatorer')/fields('%7B8d70fa93-b547-46f1-84e7-4982f8c9c675%7D') 400 (Bad Request)
> Message: Error making HttpClient request in queryable [400] Bad Request ::> {"odata.error":{"code":"-2147024809, System.ArgumentException","message":{"lang":"nb-NO","value":"Ugyldig feltnavn. {8d70fa93-b547-46f1-84e7-4982f8c9c675}  /sites/test4/Lists/Mleindikatorer"}}} Data: {"response":{},"status":400,"statusText":"Bad Request","isHttpRequestError":true}
> ```
> 
> Basically the same message with different list and fields goes on for about 60 lines after that. Then:
> 
> ```
> spfxsinglesignon.aspx:28 Cannot read property 'getRequestInfo' of null
> project-setup-application-customizer_480ec34efa912f888af570b2bc34fca5.js:26 Message: (ProjectSetup) [_initializeSetup]: Failed
> ```

None of these are relevant I'm afraid. Check if there's any details in Lists/Logg.

I just did a fresh install of 1.2.2 and provisionied 2 projects without any issues, so very curious of what might cause this.

**Comment by zetard** [comment]
Im thinking the easiest way for you is if i get you access to the tenant, its no problem since it is a developer tenant. I have sent you an invite to your email from your profile. 

URLs are /sites/prosjektportalen/ and /sites/test3/ etc or just create your own.

**Comment by olemp** [comment]
![image](https://user-images.githubusercontent.com/7606007/89174438-25849000-d586-11ea-85a1-a6965f418d2c.png)

Here's the root of the issue. Never seen something similiar. Is the tenant brand new?

**Comment by olemp** [comment]
> ![image](https://user-images.githubusercontent.com/7606007/89174438-25849000-d586-11ea-85a1-a6965f418d2c.png)
> 
> Here's the root of the issue. Never seen something similiar. Is the tenant brand new?

Wait a moment. This was without permission to the group resources it seems. Will try again.

**Comment by zetard** [comment]
The tenant is a few weeks old. It's relatively "fresh" meaning almost nothing have been done to it except setting it up.

**Comment by olemp** [comment]
> The tenant is a few weeks old. It's relatively "fresh" meaning almost nothing have been done to it except setting it up.

Seems test4 was successfully provisioned now. Did you attempt to provisioning the project with another user than the one creating the site?

**Comment by olemp** [comment]
Also succesfully provisioned **test5**.

**Comment by zetard** [comment]
No, i did everything as the same user who is also global admin etc. I'll give it a try with another new (non) admin user and report back

**Comment by olemp** [comment]
> No, i did everything as the same user who is also global admin etc. I'll give it a try with another new (non) admin user and report back

Ok, weird. I got the same error as you when attempting to provision with no access to the group resources (not being owner of the group).

**Comment by zetard** [comment]
> > No, i did everything as the same user who is also global admin etc. I'll give it a try with another new (non) admin user and report back
> 
> Ok, weird. I got the same error as you when attempting to provision with no access to the group resources (not being owner of the group).

The group resources you mention is the new group being created when creating the teamsite with "prosjektområde" as the site design, correct?

So i created a new "ordinary" user, i create a site/room as my admin user, set the "new user" as both owner and member on the new site/room. The Oppsett av prosjektområde still fails on both of the users. Any ideas?

**Comment by olemp** [comment]
> > > No, i did everything as the same user who is also global admin etc. I'll give it a try with another new (non) admin user and report back
> > 
> > 
> > Ok, weird. I got the same error as you when attempting to provision with no access to the group resources (not being owner of the group).
> 
> The group resources you mention is the new group being created when creating the teamsite with "prosjektområde" as the site design, correct?
> 
> So i created a new "ordinary" user, i create a site/room as my admin user, set the "new user" as both owner and member on the new site/room. The Oppsett av prosjektområde still fails on both of the users. Any ideas?

1. Correct
2. No idea. Works fine with the user you created for me. Did you create the new user the same way? You don't need to explicitly add anyone to members/owners group for provisioning to work.

**Comment by zetard** [comment]
User was created the same way as your user. Hmm, well i'll give it some time and look into it further, im gonna close the issue since i see it worked with your user. Will report back if i find out whats up with my users.

**Comment by olemp** [comment]
> User was created the same way as your user. Hmm, well i'll give it some time and look into it further, im gonna close the issue since i see it worked with your user. Will report back if i find out whats up with my users.

Might be a license issue?
